src/HanziWriter.js

```diff
@@ -204,11 +209,14 @@ HanziWriter.prototype._loadCharacterData = function(char) {
   this.isLoadingCharData = true;
   return new Promise((resolve, reject) => {
     this.cancelLoadingCharData = reject;
-    const returnedData = this._options.charDataLoader(char, resolve);
+    const returnedData = this._options.charDataLoader(char, resolve, reject);
     if (returnedData) resolve(returnedData);
   }).then((data) => {
     this.isLoadingCharData = false;
```

Comment:
While we're here, how about adding a `onLoadCharDataSuccess` callback here too? That way it's easy to add a loading spinner if developers want to.

Reply:
Hmm, I see... I'll do that.